packer/nomad-server/scripts/nomad.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -x
+
+set -e
+
+files=/tmp/files
```

Comment:
`readonly BASE_DIR="/tmp/files"` would be a bit more self-documenting (and safe).

Adding some descriptive comments at the top of the file describing what's going on, and the absolute dependence on the corresponding `download-files.sh` script having been run first would be great.

Links to relevant Nomad installation documentation would also be appreciated.

(All this applies equally to the Consul script, as well.)